tools/dev_console/static/index.html

```
@@ -50,18 +46,20 @@ <h1>Translation Console</h1>
       <section class="tab-panel active" id="tab-file-test">
-        <section class="workspace">
-          <aside class="panel file-list">
-            <div class="toolbar">
+
+        <section class="tool-body workspace">
+          <aside class="tool-panel panel file-list">
+            <div class="tool-header toolbar">
               <h2>Pages</h2>
               <button id="refresh" type="button">Refresh</button>
             </div>
             <input id="filter" type="search" placeholder="Filter pages">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)